spring-cloud-vault-spring-connector/src/main/java/io/pivotal/spring/cloud/vault/config/java/VaultServiceInfoPropertySourceAdapter.java

```diff
@@ -45,9 +52,23 @@
 		Map<String, Object> properties = new HashMap<String, Object>();
 
 		properties.put("spring.cloud.vault.host", serviceInfo.getHost());
-		properties.put("spring.cloud.vault.port", serviceInfo.getPort());
+		properties.put("spring.cloud.vault.port",
```

Comment:
I would suggest adding only `spring.cloud.vault.port` if `port` is not `-1` and leave defaulting up to Spring Cloud Vault. That simplifies this PR significantly and we do not need any defaulting.

Reply:
I have tried what you have suggested. When spring.cloud.vault.port is not set it will be defaulted to 8200. Result I got the following exception:

org.apache.http.conn.ConnectTimeoutException: Connect to vault-staging-lb.via.act-on.net:8200

Our Vault server is running on default HTTPS port 443 and has no port explicitly set in Vault connection URL in service broker. So port should be translated based on protocol not to Vault default.

Reply:
I see. URLs without port are always subject to some assumptions that aren't always visible to everyone. In some cases defaulting means using IANA default in other cases it's vendor-specific defaults. I'm good with these changes and I think we need to update the readme to reflect how port-less URLs impact configuration.
That is something I can do during the merge.

Please squash your commits and force-push your branch.

Reply:
Hi Mark,

I think I have squashed to much now pull request includes changes after merge from master. I do not know how to recover from that. Please advice.

Thank you,
Vasyl

Reply:
I think I was able to recover it successfully.